kube/pod_test.go

```diff
@@ -580,13 +600,16 @@ func TestPodGetEnvVarsFromConfigGenerationCounterHelm(t *testing.T) {
 		return
 	}
 	testhelpers.IsYAMLEqualString(assert, `---
-		-	name: "KUBERNETES_NAMESPACE"
-			valueFrom:
-				fieldRef:
-					fieldPath: "metadata.namespace"
-		-	name: "KUBE_SECRETS_GENERATION_COUNTER"
-			value: "3"
-	`, actual)
+- name: "KUBERNETES_NAMESPACE"
```

Comment:
I had an issue with indentation here. I didn't have time to get back to it before pushing the WIP commit (by the way, I'll rename that commit). I'll push a commit to fix it up.